plenty_api/api.py

```diff
@@ -1235,6 +1237,31 @@ def plenty_api_create_booking(self, order_id: int,
                                              data=data)
         return response
 
+
+    def plenty_api_create_property_selection_name(self, json: dict) -> dict:
+        """
+        Create a language for a selection property.
```

Comment:
Wow you create a language :O .. jokes aside, s/language/name in a specific language/